### ISSUE TITLE
【EESS】New submissions for Fri, 12 Jan 24

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#1083]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imaging
There is no result 
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
### Multi-frequency antenna for quasi-isotropic radiator and 6G massive IoT
 - **Authors:** Bing Xiao, Hang Wong, Kam Man Shum
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2401.05383
 - **Pdf link:** https://arxiv.org/pdf/2401.05383
 - **Abstract**
 An isotropic antenna radiates and receives electromagnetic wave uniformly in magnitude in 3D space. A multi-frequency quasi-isotropic antenna can serve as a practically feasible solution to emulate an ideal multi-frequency isotropic radiator. It is also an essential technology for mobile smart devices for massive IoT in the upcoming 6G. However, ever since the quasi-isotropic antenna was proposed and achieved more than half a century ago, at most two discrete narrow frequency bands can be achieved, because of the significantly increased structural complexity from multi-frequency isotropic radiation. This limitation impedes numerous related electromagnetic experiments and the advances in wireless communication. Here, for the first time, a design method for multi-band (>2) quasi-isotropic antennas is proposed. An exemplified quasi-isotropic antenna with the desired four frequency bands is also presented for demonstration. The measured results validate excellent performance on both electromagnetics and wireless communications for this antenna.
### Decoding Emotional Valence from Wearables: Can Our Data Reveal Our True  Feelings?
 - **Authors:** Michal K. Grzeszczyk, Anna Lisowska, Arkadiusz Sitek, Aneta Lisowska
 - **Subjects:** Signal Processing (eess.SP); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2401.05408
 - **Pdf link:** https://arxiv.org/pdf/2401.05408
 - **Abstract**
 Automatic detection and tracking of emotional states has the potential for helping individuals with various mental health conditions. While previous studies have captured physiological signals using wearable devices in laboratory settings, providing valuable insights into the relationship between physiological responses and mental states, the transfer of these findings to real-life scenarios is still in its nascent stages. Our research aims to bridge the gap between laboratory-based studies and real-life settings by leveraging consumer-grade wearables and self-report measures. We conducted a preliminary study involving 15 healthy participants to assess the efficacy of wearables in capturing user valence in real-world settings. In this paper, we present the initial analysis of the collected data, focusing primarily on the results of valence classification. Our findings demonstrate promising results in distinguishing between high and low positive valence, achieving an F1 score of 0.65. This research opens up avenues for future research in the field of mobile mental health interventions.
### An Unobtrusive and Lightweight Ear-worn System for Continuous Epileptic  Seizure Detection
 - **Authors:** Abdul Aziz, Nhat Pham, Neel Vora, Cody Reynolds, Jaime Lehnen, Pooja Venkatesh, Zhuoran Yao, Jay Harvey, Tam Vu, Kan Ding, Phuc Nguyen
 - **Subjects:** Signal Processing (eess.SP); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2401.05425
 - **Pdf link:** https://arxiv.org/pdf/2401.05425
 - **Abstract**
 Epilepsy is one of the most common neurological diseases globally, affecting around 50 million people worldwide. Fortunately, up to 70 percent of people with epilepsy could live seizure-free if properly diagnosed and treated, and a reliable technique to monitor the onset of seizures could improve the quality of life of patients who are constantly facing the fear of random seizure attacks. The scalp-based EEG test, despite being the gold standard for diagnosing epilepsy, is costly, necessitates hospitalization, demands skilled professionals for operation, and is discomforting for users. In this paper, we propose EarSD, a novel lightweight, unobtrusive, and socially acceptable ear-worn system to detect epileptic seizure onsets by measuring the physiological signals from behind the user's ears. EarSD includes an integrated custom-built sensing, computing, and communication PCB to collect and amplify the signals of interest, remove the noises caused by motion artifacts and environmental impacts, and stream the data wirelessly to the computer or mobile phone nearby, where data are uploaded to the host computer for further processing. We conducted both in-lab and in-hospital experiments with epileptic seizure patients who were hospitalized for seizure studies. The preliminary results confirm that EarSD can detect seizures with up to 95.3 percent accuracy by just using classical machine learning algorithms.
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
### Online Action Recognition for Human Risk Prediction with Anticipated  Haptic Alert via Wearables
 - **Authors:** Cheng Guo (1 and 2), Lorenzo Rapetti (1), Kourosh Darvish (3), Riccardo Grieco (1), Francesco Draicchio (4), Daniele Pucci (1 and 2) ((1) Istituto Italiano di Tecnologia, (2) University of Manchester, (3) University of Toronto, (4) INAIL)
 - **Subjects:** Signal Processing (eess.SP); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2401.05365
 - **Pdf link:** https://arxiv.org/pdf/2401.05365
 - **Abstract**
 This paper proposes a framework that combines online human state estimation, action recognition and motion prediction to enable early assessment and prevention of worker biomechanical risk during lifting tasks. The framework leverages the NIOSH index to perform online risk assessment, thus fitting real-time applications. In particular, the human state is retrieved via inverse kinematics/dynamics algorithms from wearable sensor data. Human action recognition and motion prediction are achieved by implementing an LSTM-based Guided Mixture of Experts architecture, which is trained offline and inferred online. With the recognized actions, a single lifting activity is divided into a series of continuous movements and the Revised NIOSH Lifting Equation can be applied for risk assessment. Moreover, the predicted motions enable anticipation of future risks. A haptic actuator, embedded in the wearable system, can alert the subject of potential risk, acting as an active prevention device. The performance of the proposed framework is validated by executing real lifting tasks, while the subject is equipped with the iFeel wearable system.
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Generalizable Sleep Staging via Multi-level Domain Alignment
 - **Authors:** Jiquan Wang, Sha Zhao, Haiteng Jiang, Shijian Li, Tao Li, Gang Pan
 - **Subjects:** Signal Processing (eess.SP); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2401.05363
 - **Pdf link:** https://arxiv.org/pdf/2401.05363
 - **Abstract**
 Automatic sleep staging is essential for sleep assessment and disorder diagnosis. Most existing methods depend on one specific dataset and are limited to be generalized to other unseen datasets, for which the training data and testing data are from the same dataset. In this paper, we introduce domain generalization into automatic sleep staging and propose the task of generalizable sleep staging which aims to improve the model generalization ability to unseen datasets. Inspired by existing domain generalization methods, we adopt the feature alignment idea and propose a framework called SleepDG to solve it. Considering both of local salient features and sequential features are important for sleep staging, we propose a Multi-level Feature Alignment combining epoch-level and sequence-level feature alignment to learn domain-invariant feature representations. Specifically, we design an Epoch-level Feature Alignment to align the feature distribution of each single sleep epoch among different domains, and a Sequence-level Feature Alignment to minimize the discrepancy of sequential features among different domains. SleepDG is validated on five public datasets, achieving the state-of-the-art performance.
### ADF & TransApp: A Transformer-Based Framework for Appliance Detection  Using Smart Meter Consumption Series
 - **Authors:** Adrien Petralia, Philippe Charpentier, Themis Palpanas
 - **Subjects:** Signal Processing (eess.SP); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2401.05381
 - **Pdf link:** https://arxiv.org/pdf/2401.05381
 - **Abstract**
 Over the past decade, millions of smart meters have been installed by electricity suppliers worldwide, allowing them to collect a large amount of electricity consumption data, albeit sampled at a low frequency (one point every 30min). One of the important challenges these suppliers face is how to utilize these data to detect the presence/absence of different appliances in the customers' households. This valuable information can help them provide personalized offers and recommendations to help customers towards the energy transition. Appliance detection can be cast as a time series classification problem. However, the large amount of data combined with the long and variable length of the consumption series pose challenges when training a classifier. In this paper, we propose ADF, a framework that uses subsequences of a client consumption series to detect the presence/absence of appliances. We also introduce TransApp, a Transformer-based time series classifier that is first pretrained in a self-supervised way to enhance its performance on appliance detection tasks. We test our approach on two real datasets, including a publicly available one. The experimental results with two large real datasets show that the proposed approach outperforms current solutions, including state-of-the-art time series classifiers applied to appliance detection. This paper appeared in VLDB 2024.
### CoSS: Co-optimizing Sensor and Sampling Rate for Data-Efficient AI in  Human Activity Recognition
 - **Authors:** Mengxi Liu, Zimin Zhao, Daniel Geißler, Bo Zhou, Sungho Suh, Paul Lukowicz
 - **Subjects:** Signal Processing (eess.SP); Artificial Intelligence (cs.AI); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2401.05426
 - **Pdf link:** https://arxiv.org/pdf/2401.05426
 - **Abstract**
 Recent advancements in Artificial Neural Networks have significantly improved human activity recognition using multiple time-series sensors. While employing numerous sensors with high-frequency sampling rates usually improves the results, it often leads to data inefficiency and unnecessary expansion of the ANN, posing a challenge for their practical deployment on edge devices. Addressing these issues, our work introduces a pragmatic framework for data-efficient utilization in HAR tasks, considering the optimization of both sensor modalities and sampling rate simultaneously. Central to our approach are the designed trainable parameters, termed 'Weight Scores,' which assess the significance of each sensor modality and sampling rate during the training phase. These scores guide the sensor modalities and sampling rate selection. The pruning method allows users to make a trade-off between computational budgets and performance by selecting the sensor modalities and sampling rates according to the weight score ranking. We tested our framework's effectiveness in optimizing sensor modality and sampling rate selection using three public HAR benchmark datasets. The results show that the sensor and sampling rate combination selected via CoSS achieves similar classification performance to configurations using the highest sampling rate with all sensors but at a reduced hardware cost.
### TRLS: A Time Series Representation Learning Framework via Spectrogram  for Medical Signal Processing
 - **Authors:** Luyuan Xie, Cong Li, Xin Zhang, Shengfang Zhai, Yuejian Fang, Qingni Shen, Zhonghai Wu
 - **Subjects:** Signal Processing (eess.SP); Artificial Intelligence (cs.AI); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2401.05431
 - **Pdf link:** https://arxiv.org/pdf/2401.05431
 - **Abstract**
 Representation learning frameworks in unlabeled time series have been proposed for medical signal processing. Despite the numerous excellent progresses have been made in previous works, we observe the representation extracted for the time series still does not generalize well. In this paper, we present a Time series (medical signal) Representation Learning framework via Spectrogram (TRLS) to get more informative representations. We transform the input time-domain medical signals into spectrograms and design a time-frequency encoder named Time Frequency RNN (TFRNN) to capture more robust multi-scale representations from the augmented spectrograms. Our TRLS takes spectrogram as input with two types of different data augmentations and maximizes the similarity between positive ones, which effectively circumvents the problem of designing negative samples. Our evaluation of four real-world medical signal datasets focusing on medical signal classification shows that TRLS is superior to the existing frameworks.
### Self-supervised Learning for Electroencephalogram: A Systematic Survey
 - **Authors:** Weining Weng, Yang Gu, Shuai Guo, Yuan Ma, Zhaohua Yang, Yuchen Liu, Yiqiang Chen
 - **Subjects:** Signal Processing (eess.SP); Artificial Intelligence (cs.AI); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2401.05446
 - **Pdf link:** https://arxiv.org/pdf/2401.05446
 - **Abstract**
 Electroencephalogram (EEG) is a non-invasive technique to record bioelectrical signals. Integrating supervised deep learning techniques with EEG signals has recently facilitated automatic analysis across diverse EEG-based tasks. However, the label issues of EEG signals have constrained the development of EEG-based deep models. Obtaining EEG annotations is difficult that requires domain experts to guide collection and labeling, and the variability of EEG signals among different subjects causes significant label shifts. To solve the above challenges, self-supervised learning (SSL) has been proposed to extract representations from unlabeled samples through well-designed pretext tasks. This paper concentrates on integrating SSL frameworks with temporal EEG signals to achieve efficient representation and proposes a systematic review of the SSL for EEG signals. In this paper, 1) we introduce the concept and theory of self-supervised learning and typical SSL frameworks. 2) We provide a comprehensive review of SSL for EEG analysis, including taxonomy, methodology, and technique details of the existing EEG-based SSL frameworks, and discuss the difference between these methods. 3) We investigate the adaptation of the SSL approach to various downstream tasks, including the task description and related benchmark datasets. 4) Finally, we discuss the potential directions for future SSL-EEG research.
### Robust CNN-based Respiration Rate Estimation for Smartwatch PPG and IMU
 - **Authors:** Kianoosh Kazemi, Iman Azimi, Pasi Liljeberg, Amir M. Rahmani
 - **Subjects:** Signal Processing (eess.SP); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2401.05469
 - **Pdf link:** https://arxiv.org/pdf/2401.05469
 - **Abstract**
 Respiratory rate (RR) serves as an indicator of various medical conditions, such as cardiovascular diseases and sleep disorders. These RR estimation methods were mostly designed for finger-based PPG collected from subjects in stationary situations (e.g., in hospitals). In contrast to finger-based PPG signals, wrist-based PPG are more susceptible to noise, particularly in their low frequency range, which includes respiratory information. Therefore, the existing methods struggle to accurately extract RR when PPG data are collected from wrist area under free-living conditions. The increasing popularity of smartwatches, equipped with various sensors including PPG, has prompted the need for a robust RR estimation method. In this paper, we propose a convolutional neural network-based approach to extract RR from PPG, accelerometer, and gyroscope signals captured via smartwatches. Our method, including a dilated residual inception module and 1D convolutions, extract the temporal information from the signals, enabling RR estimation. Our method is trained and tested using data collected from 36 subjects under free-living conditions for one day using Samsung Gear Sport watches. For evaluation, we compare the proposed method with four state-of-the-art RR estimation methods. The RR estimates are compared with RR references obtained from a chest-band device. The results show that our method outperforms the existing methods with the Mean-Absolute-Error and Root-Mean-Square-Error of 1.85 and 2.34, while the best results obtained by the other methods are 2.41 and 3.29, respectively. Moreover, compared to the other methods, the absolute error distribution of our method was narrow (with the lowest median), indicating a higher level of agreement between the estimated and reference RR values.
### End-to-End Learning for SLP-Based ISAC Systems
 - **Authors:** Yixian Zheng, Rang Liu, Ming Li, Qian Liu
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2401.05663
 - **Pdf link:** https://arxiv.org/pdf/2401.05663
 - **Abstract**
 Integrated sensing and communication (ISAC) is an encouraging wireless technology which can simultaneously perform both radar and communication functionalities by sharing the same transmit waveform, spectral resource, and hardware platform. Recently emerged symbol-level precoding (SLP) technique exhibits advancement in ISAC systems by leveraging the waveform design degrees of freedom (DoFs) in both temporal and spatial domains. However, traditional SLP-based ISAC systems are designed in a modular paradigm, which potentially limits the overall performance of communication and radar sensing. The high complexity of existing SLP design algorithms is another issue that hurdles the practical deployment. To break through the bottleneck of these approaches, in this paper we propose an end-to-end approach to jointly design the SLP-based dual-functional transmitter and receivers of communication and radar sensing. In particular, we aim to utilize deep learning-based methods to minimize the symbol error rate (SER) of communication users, maximize the detection probability, and minimize the root mean square error (RMSE) of the target angle estimation. Multi-layer perceptron (MLP) networks and a long short term memory (LSTM) network are respectively applied to the transmitter, communication users and radar receiver. Simulation results verify the feasibility of the proposed deep-learning-based end-to-end optimization for ISAC systems and reveal the effectiveness of the proposed neural networks for the end-to-end design.
### Neural Implicit Surface Reconstruction for Freehand 3D Ultrasound  Volumetric Point Clouds with Geometric Constraints
 - **Authors:** Hongbo Chen, Logiraj Kumaralingam, Edmond H. M. Lou, Kumaradevan Punithakumar, Jiawen Li, Thanh-Tu Pham, Lawrence H. Le, Rui Zheng
 - **Subjects:** Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2401.05915
 - **Pdf link:** https://arxiv.org/pdf/2401.05915
 - **Abstract**
 Three-dimensional (3D) freehand ultrasound (US) is a widely used imaging modality that allows non-invasive imaging of medical anatomy without radiation exposure. The freehand 3D US surface reconstruction is vital to acquire the accurate anatomical structures needed for modeling, registration, and visualization. However, the currently used traditional methods cannot produce a high-quality surface due to imaging noise and connectivity issues in US. Although the deep learning-based approaches exhibiting the improvements in smoothness, continuity and resolution, the investigation into freehand 3D US remains limited. In this study, we introduce a self-supervised neural implicit surface reconstruction method to learn the signed distance functions (SDFs) from freehand 3D US volumetric point clouds. In particular, our method iteratively learns the SDFs by moving the 3D queries sampled around the point clouds to approximate the surface with the assistance of two novel geometric constraints. We assess our method on the three imaging systems, using twenty-three shapes that include six distinct anthropomorphic phantoms datasets and seventeen in vivo carotid artery datasets. Experimental results on phantoms outperform the existing approach, with a 67% reduction in Chamfer distance, 60% in Hausdorff distance, and 61% in Average absolute distance. Furthermore, our method achieves a 0.92 Dice score on the in vivo datasets and demonstrates great clinical potential.
